sdk/vaa/structs.go

```diff
@@ -444,6 +444,8 @@ func (v *VAA) VerifySignatures(addresses []common.Address) bool {
 
 	return true
 }
+
+// Takes the guardian set keys and checks this VAA for signatures and quorum
```

Comment:
Being super nit-picky here but let's make sure we follow to godoc guidelines: https://go.dev/blog/godoc

This means comments on functions and structs should start with the name of the func or struct being described.

`Verify takes the guardian set keys and checks this VAA for signatures and quorum`

Also let's be a little more specific in what it actually does. Does it check for signatures or verify the signatures given the provided guardian set.

E.g. `Verify verifies the signatures on the VAA using the provided guardian set keys and checks for quorum. It returns an error if any verification step fails.`

Reply:
Correct go documentation attempted.